### PR TITLE
Remove insiders in peerDependencies

### LINKED ISSUE #325: Peer dependency invalid version requirement
### What version of @tailwindcss/typography are you using?

v0.5.8

### What version of Node.js are you using?

v20.5.0

### What browser are you using?

n/a

### What operating system are you using?

macOS

### Reproduction repository

https://semver.npmjs.com/

### Describe your issue

1. Go to https://semver.npmjs.com/
2. Set the package name to `@tailwindcss/typography`
3. and version range to `>=3.0.0 || insiders` (as defined [here](https://github.com/tailwindlabs/tailwindcss-typography/blob/a86e6015694c3435ff6cef84f3dd61b81adf26e1/package.json#L31))

### Expected

A valid range

### Actual result

```
✕ Invalid range. View [syntax examples](https://semver.npmjs.com/#syntax-examples)
```

### Notes

This breaks npm updates, here is an example using `npm-check-updates` 

```
npm-check-updates --peer --target minor   
Checking /Users/lil5/Projects/clothingloop/clothing-loop/frontend/package.json
[====================] 51/51 100%
[====================] 38/38 100%

 @tailwindcss/typography              ^0.5.8  →   ^0.5.10
 autoprefixer                       ^10.4.13  →  ^10.4.16

Ignored incompatible updates (peer dependencies):

 tailwindcss  ^3.3.0  →  ^3.3.5  reason: @tailwindcss/typography requires >=3.0.0 || insiders

Run ncu --peer --target minor -u to upgrade package.json
```

### PR DESCRIPTION
**Pull request by lil5**
Fixes #325

### CODE REVIEW

**Comment by thecrypticace** [comment]
See https://github.com/tailwindlabs/tailwindcss-typography/issues/325#issuecomment-1792652129

TL;DR this is not invalid
